packages/fast-development-site-react/src/components/toc/toc-item.tsx

```diff
@@ -13,22 +13,44 @@ export interface ITocItemProps {
 }
 
 export interface ITocItemManagedClasses {
+    a: string;
     toc__item: string;
     toc__item_active: string;
 }
 
 export enum itemType {
+    a = "a",
     tocMenu = "tocMenu",
     link = "link"
 }
 
+// tslint:disable-next-line
+const activeItemPipe: string = "PHN2ZyB3aWR0aD0iNCIgaGVpZ2h0PSIyMSIgdmlld0JveD0iMCAwIDQgMjEiIHZlcnNpb249IjEuMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayI+DQo8dGl0bGU+QWN0aXZlIGl0ZW0gcGlwZTwvdGl0bGU+DQo8ZGVzYz5DcmVhdGVkIHVzaW5nIEZpZ21hPC9kZXNjPg0KPGcgaWQ9IkNhbnZhcyIgdHJhbnNmb3JtPSJ0cmFuc2xhdGUoNDc0NiAxMTMxKSI+DQo8ZyBpZD0iQWN0aXZlIGl0ZW0gcGlwZSI+DQo8dXNlIHhsaW5rOmhyZWY9IiNwYXRoMF9maWxsIiB0cmFuc2Zvcm09InRyYW5zbGF0ZSgtNDc0NiAtMTEzMSkiIGZpbGw9IiMwMDc4RDciLz4NCjwvZz4NCjwvZz4NCjxkZWZzPg0KPHBhdGggaWQ9InBhdGgwX2ZpbGwiIGQ9Ik0gNCAxOC44NTc1TCA0IDJDIDQgMC44OTU0MzEgMy4xMDQ1NyAwIDIgMEMgMC44OTU0MzEgMCAwIDAuODk1NDMxIDAgMkwgMCAxOC44NTc1QyAwIDE5Ljk2MjEgMC44OTU0MzEgMjAuODU3NSAyIDIwLjg1NzVDIDMuMTA0NTcgMjAuODU3NSA0IDE5Ljk2MjEgNCAxOC44NTc1WiIvPg0KPC9kZWZzPg0KPC9zdmc+DQo=";
+
 const style: ComponentStyles<ITocItemManagedClasses, IDevSiteDesignSystem> = {
+    a: {
+        color: "#000000",
+        textDecoration: "none",
+        display: "block",
+        textIndent: "48px",
+        lineHeight: "40px",
+        outline: "0"
+    },
     toc__item: {
-        padding: "6px 12px",
-        display: "block"
+        display: "block",
+        position: "relative",
+        color: "#000000"
     },
     toc__item_active: {
-        background: "lightgray"
+        background: "#FFFFFF",
+        color: "#000000",
+        boxShadow: "0px 2px 4px -1px #ccc;",
+        "&::before": {
+            content: `url('data:image/svg+xml;base64,${activeItemPipe}')`,
```

Comment:
more of an FYI - you can move `data:image/svg+xml;base64,` directly into the string def